compiler/grainformat/reformat.re

```diff
@@ -108,12 +108,68 @@ let get_original_code = (location: Location.t, source: array(string)) => {
   };
 };
 
+// Be AWARE!  This is only to be called when you know the comments list is not empty.
+// Moved here in case we want to change the implementation in future
+let get_last_item_in_list = comments =>
+  List.nth(comments, List.length(comments) - 1);
+
+let is_disable_formatting_comment = (comment: Parsetree.comment) => {
+  switch (comment) {
+  | Line(cmt) =>
+    if (String.trim(Comments.get_comment_source(comment))
```

Comment:
Why are you using `get_comment_source`? The cmt already has a trimmed property.

Reply:
You are not wrong.  Fixed